src/Security/Member.php

```diff
@@ -1625,17 +1625,17 @@ public function validate(): ValidationResult
             return ValidationResult::create();
         }
 
-        $valid = parent::validate();
+        $result = parent::validate();
```

Comment:
Unrelated refactoring to use the standard `$result` variable we use everywhere else for a `ValidationResult`